README.md

```diff
@@ -32,6 +32,12 @@ Running the docker image will save artifacts into `$(pwd)/build`:
 $ docker run --rm -v $(pwd):/irods_docs:ro -v $(pwd)/build:/hostcomputer irods_docs_builder
 ```
 
+You can generate documentation for a specific build of iRODS by running the following:
+
+```
+$ docker run --rm -v $(pwd):/irods_docs:ro -v $(pwd)/build:/hostcomputer irods_docs_builder MAKEGITHUBACCOUNT=<account> MAKEIRODSVERSION=<branch-or-commit>
```

Comment:
take it or leave it...  consider...  MAKEIRODSVERSION -> MAKEGITCOMMITTISH ?

Reply:
How do you feel about removing the `MAKE` prefix and adding underscores?
- `GITHUB_ACCOUNT`
- `GIT_COMMITTISH`

Reply:
agreed, those seem better all around.

Reply:
done